igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/model/GenericBuild.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.build.model
+
+class GenericBuild {
```

Comment:
I'm wondering if Build is the best name here, since CI servers to do more than just produce artifacts (and Spinnaker uses them for more than just that). WDYT @tomaslin @duftler?

Also, I don't think you need to prefix this class or GenericArtifact with "Generic"